static/app/views/insights/http/views/httpDomainSummaryPage.tsx

```diff
@@ -292,6 +293,7 @@ export function HTTPDomainSummaryPage() {
                 series={[durationData[`avg(${SpanMetricsField.SPAN_SELF_TIME})`]]}
                 isLoading={isDurationDataLoading}
                 error={durationError}
+                filters={filters}
```

Comment:
Long term, it would be cool if we didn't have to pass in filters into every single chart, and instead determine the filters based on the module + url.